app/models/concerns/mets_structure.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module MetsStructure
+  # def structure
+  #   structure_type('logical')
+  # end
+  #
+  # def structure_for_volume(volume_id)
+  #   volume = volume_nodes.find { |vol| vol.attribute("ID").value == volume_id }
```

Comment:
where does the term "volume" come from?  I wonder if we want something more generic or specifically from METS here?  I think "volume" might carry semantic weight that we don't want here - i.e. a volume for Beinecke is one specific type of artifact and we'll have lots of different things being described.